.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "sqltools.connections": [
+        {
+            "previewLimit": 50,
+            "server": "localhost",
+            "port": 5432,
+            "driver": "PostgreSQL",
+            "name": "new",
+            "group": "new",
+            "database": "ecommerce_db",
+            "username": "postgres"
+        }
+    ]
+}
```

Comment:
please pay attention

Reply:
added .vscode in .gitignore and removed it